src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-details/rgw-storage-class-details.component.html

```diff
@@ -0,0 +1,96 @@
+<ng-container *ngIf="!!selection">
+  <cds-tabs type="contained"
+            theme="light">
+    <cds-tab heading="Details">
+      <table
+        class="cds--data-table--sort cds--data-table--no-border cds--data-table cds--data-table--md"
+        data-testid="rgw-storage-details"
+      >
+        <tbody>
+          <tr>
+            <td class="bold">
+              Target Path
+              <cd-helper class="text-pre-wrap"
+                         i18n>
```

Comment:
```suggestion
```
You should be adding i18n on the first parent (closest to text). So here we can add to span